Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN apt-get -q update \
     libpng12-0 \
     libx11-6 \
     libxpm4 \
-    mysql-client
+    mysql-client \
+    postgresql-client libpq-dev
```

Comment:
okay, didn't recognise how the list had been sorted, so I just put both postgres deps on the same line.